msi/GitProduct.wxs

```diff
@@ -158,6 +167,11 @@
       <Directory Id='ProgramFiles$(var.SixtyFourBit)Folder' Name='PFiles'>
         <Directory Id='INSTALLFOLDER' Name='Git'>
           <Directory Id='BinFolder' Name='bin' />
+        <?if $(var.SixtyFourBit)=64 ?>
+          <Directory Id='MingwFolder' Name='mingw64' />
+        <?else?>
+          <Directory Id='MingwFolder' Name='mingw32' />
+        <?endif?>
```

Comment:
Would it not make more sense to generate this via `release.sh`, which already knows which one is correct?